Source/API/EbApi.h

```diff
@@ -138,6 +145,7 @@ typedef struct EB_H265_ENC_CONFIGURATION
     int32_t                 frameRateNumerator;
     int32_t                 frameRateDenominator;
     uint32_t                encoderBitDepth;
+    EB_COLOR_FORMAT         colorFormat;
```

Comment:
where is it used?

Reply:
I'm enabling 422/444 support, so it will be used for following patches

Reply:
where are the following patches? Since it's related, the PR can still be used to add the feature.